frontend/packages/ux-editor/src/components/config/editModal/EditGrid.tsx

```diff
@@ -0,0 +1,85 @@
+import React, { useState, ReactNode } from 'react';
+import type { IGenericEditComponent } from '../componentConfig';
+import { useText } from '../../../hooks';
+import { Tabs } from '@digdir/design-system-react';
+import classes from './EditGrid.module.css';
+import { EditGridForGivenViewSize } from './EditGridForGivenViewSize';
+import { LaptopIcon, MobileSmallIcon } from '@navikt/aksel-icons';
+import { FormComponent } from '../../../types/FormComponent';
+import { deepCopy } from 'app-shared/pure';
+
+export enum ViewSizeForGridProp {
+  S = 'xs',
+  M = 'md',
+}
+
+export interface GridSizeForViewSize {
+  xs?: number;
+  md?: number;
+}
+
+const getIconForViewSize = (viewSize: ViewSizeForGridProp): ReactNode => {
+  const iconMapping = {
+    [ViewSizeForGridProp.S]: <MobileSmallIcon />,
+    [ViewSizeForGridProp.M]: <LaptopIcon />,
+  };
+  return iconMapping[viewSize] || null;
+};
```

Comment:
How about directly implementing the hash lookup instead of creating a separate method that needs to be invoked?

```suggestion
  const iconMapping: Record<ViewSizeForGridProp, ReactNode> = {
    [ViewSizeForGridProp.S]: <MobileSmallIcon />,
    [ViewSizeForGridProp.M]: <LaptopIcon />,
  };
  ```
```
  <Tabs.Tab key={viewSize} value={viewSize} icon={iconMapping[viewSize] || null}>
```